.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+#### Pull Request Checklist ####
+
+- [ ] Make sure the `packageVersion` in the `package.yaml` file of the chart you are modifying is at least one version higher than the `packageVersion` of your chart in https://github.com/rancher/charts/tree/main/assets to avoid overwriting the existing chart archive.
```

Comment:
Maybe worth noting: for original chart(do not have package.yaml), developer should update the version in `Chart.yaml`. Also it is only a hard requirement to not overwriting charts that has been published in `main-source` branch.

Reply:
good catches, updated the PR template accordingly in the latest push